options/dopex/clamm.ts

```diff
@@ -88,9 +92,11 @@ async function getChainStats({ graphUrl, timestamp }: IGetChainStatsParams) {
   return {
     timestamp,
     ...cumulative,
-    totalFees: cumulative.totalRevenue,
     ...daily,
```

Comment:
fees is require `dailyFees` we can't list without dailyFees

Reply:
In context of options, this will be protocol fees + premium?

when a user purchases options on stryke they pay premium + protocol fees